benches/geninput.py

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env python3
-from array import array
-from itertools import product as cartesian_product
-from random import Random
-from os.path import join, exists, dirname, realpath
-from os import environ
-from fractions import Fraction as frac
+from array          import array
+from itertools      import product as cartesian_product
+from random         import Random
+from os.path        import join, exists, dirname, realpath
+from os             import environ
 import sys
 
+sys.path.append(dirname(realpath(__file__)))
+
+from fractions import Fraction as frac
+
```

Comment:
Those lines should not be changed:
1. Python pep8 do not want additional spaces after 'from x' and 'import'
2. Fractions module is installed using pip